docs/topics/proc_interacting-with-the-secured-booster-on-the-command-line.adoc

```diff
@@ -1,6 +1,6 @@
 
-[#interacting-with-the-secured-booster-on-the-command-line]
-= Interacting with the {name-mission-secured} Booster on the Command Line
+[#authenticating-with-the-secured-booster-using-the-command-line]
+= Authenticating with the {name-mission-secured} Booster using the Command Line
```

Comment:
Not a fan of using _with_ here. From what I gathered around the internet, you authenticate _to_ (or _against_) a service _with_ credentials.